plugins/backend/catalog-backend-module-okta/src/providers/OktaOrgEntityProvider.ts

```diff
@@ -171,14 +178,30 @@ export class OktaOrgEntityProvider extends OktaEntityProvider {
               group,
               oktaGroups,
             });
-
+            const profileAnnotations: Record<string, string> = {};
+            if (this.customAttributesToAnnotationAllowlist.length) {
```

Comment:
There is no need to check the length here, you are already looping over the list. Not checking should be equivilent.